frontend-pages/profile-page.php

```diff
@@ -136,7 +139,25 @@ function ssv_profile_page_content_tabs()
         <?php
     }
     ?>
-    <a href="?view=pdf" class="mui-btn mui-btn--primary button-primary">Print Profile</a>
+    <a id="pdf-profile-button" href="#" class="mui-btn mui-btn--primary button-primary">PDF Profile</a>
+    <script>
+        var button = $('#pdf-profile-button');
+        button.removeAttr("onclick");
+        button.click(function(){
+            <?php
+            $url = get_permalink();
+            $url .= '?view=pdf';
+            if (isset($_GET['user_id'])) {
+                $url .= '&user_id=';
+                $url .= $_GET['user_id'];
+            }
+            ?>
+            $.ajax({url: '<?php echo $url; ?>', success: function(data){
+                window.open('<?php echo get_site_url() . '/wp-content/plugins/ssv-frontend-members/exported-profiles/' . str_replace(' ', '_', $member->display_name . '.pdf'); ?>');
```

Comment:
Why not show the PDF directly instead of creating a popup which most decent browsers block? People can just use the shiny "back" button in their browser to go back.